client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -345,6 +347,39 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
```

Comment:
Ah, I see - makes sense 👍 Then my only concern is that someone could add this rule to the domains list view at some point in time and then it won't behave as one might think. Maybe a comment warning about that would suffice here (though, who reads comments? 😝 ).